.vscode/launch.json

```diff
@@ -32,6 +32,17 @@
             "mainClass": "ded_card.DedCardApplication",
             "projectName": "ded_card",
             "args": "",
+            "envFile": "${workspaceFolder}/.env",
```

Comment:
co to jest za dodatkowy kod? Dodaje funkcje jsonowi?

Reply:
dobre pytanie, wystartoowalem z twojego brancha i juz tam to bylo